src/storage/exec/EdgeNode.h

```diff
@@ -126,10 +126,10 @@ class FetchEdgeNode final : public EdgeNode<cpp2::EdgeKey> {
         }
         prefix_ = NebulaKeyUtils::edgePrefix(planContext_->vIdLen_,
                                              partId,
-                                             edgeKey.src,
+                                             edgeKey.src.getStr(),
                                              edgeKey.edge_type,
                                              edgeKey.ranking,
-                                             edgeKey.dst);
```

Comment:
Have a doubt.
You have changed a lot of member functions of `NebulaKeyUtils`,
but have not seen the modification of the definition of these member functions?

Reply:
Emm, I didn't change the utils itself.